### ISSUE TITLE
App Crashes while clicking on the area just above the eye icon in sign in

### ISSUE THREAD

**Issue by asaikarthikeya** [enviroCar/enviroCar-app#842]
Click on the background just above the eye icon, On the area where background contains Munster name.
Same can be observed in sign up. Click on area where background contains Munster text, App crashes.

https://user-images.githubusercontent.com/70392921/136885819-7c340276-6131-4376-8641-e5ded2a7b72b.mp4

**Comment by asaikarthikeya** [comment]
@SebaDro I just found out that this happens with the whole background. Clicking on background where names of cities are present causes app crach. Are u able to reproduce this sir?

**Comment by cdhiraj40** [comment]
> Very weird^^. Would you like to work on it?

I would sure like to work but from past days I just can't build the files because of 
``` Cannot resolve symbol for dagger component ```
I have rebuild, cleaned, invalidate & restart but nothing seems to work :/

**Comment by cdhiraj40** [comment]
> That's strange, since the last CI build successfully finished, yesterday: https://github.com/enviroCar/enviroCar-app/actions/runs/1337266143

I have talked with @asaikarthikeya about this too and it seems like it's a problem from my side only :/

**Comment by asaikarthikeya** [comment]
@SebaDro We the current implementation we have onclick for the background to close the keyboard
**@OnClick(R.id.signin_background)
    protected void closeKeyboard(){
        InputMethodManager imm = (InputMethodManager)getSystemService(Context.INPUT_METHOD_SERVICE);
        imm.hideSoftInputFromWindow(getCurrentFocus().getWindowToken(), 0);
    }**

This method gives Null pointer exception when the edit text is on in focus. 

Possible solution - https://stackoverflow.com/questions/7940765/how-to-hide-the-soft-keyboard-inside-a-fragment

**Comment by asaikarthikeya** [comment]
> > That's strange, since the last CI build successfully finished, yesterday: https://github.com/enviroCar/enviroCar-app/actions/runs/1337266143
> 
> I have talked with @asaikarthikeya about this too and it seems like it's a problem from my side only :/

Ya..Kindly follow the steps that i have mentioned to you in the personal Mail. That might work. Thanks :)

**Comment by SebaDro** [comment]
Ok, I understand that the dispatchEvent approach has advantages when it comes to handle different fragments in one activity. However, we have a working and stable solution in those fragments that needs to hide the keyboard when clicking outside the editText field. In addition, you second code sample seems very overelaborated for implementing such a simple action. 

Therefore, I'd prefer the existing approach because it is more straightforward. So I'd suggest to only add a `null` check for the view you want to hide the keyboard from.

**Comment by SebaDro** [comment]
Yeah, do it 😃

**Comment by SebaDro** [comment]
Yes, I can confirm

**Comment by SebaDro** [comment]
Fixed by #846

**Comment by cdhiraj40** [comment]
@SebaDro @asaikarthikeya can we create an issue for just discussion about adding new features or simply for comments on the app?
I wanted to propose --> issue template, PR template, and many more :)

**Comment by asaikarthikeya** [comment]
@cdhiraj40 Hello Dhiraj, Things such as PR template,issue template are handled by the internal org members. Not open to external contributors at this moment. We would definetly like to here your new ideas, so donot hesistate to create an issue if you have a new feature/ bug in your mind. Work only after it is approved by any mentor ;)

**Comment by SebaDro** [comment]
Hi @cdhiraj40.
We introduced our new contributing guidelines in the develop branch: https://github.com/enviroCar/enviroCar-app/blob/develop/CONTRIBUTING.md

Have a look on it, there are also some basic guidelines for reporting issues and creating pull requests. In the future, maybe, we will also allow discussions about new ideas using GitHub Discussions. But up to now, however, we did not activate it, yet.
